tests/Console/ConfigurationResolverTest.php

```diff
@@ -961,6 +961,26 @@ public function provideResolveRulesCases()
                     '@A' => true,
                 ),
             ),
+            '@A - @C' => array(
```

Comment:
@julienfalque , but then it would be like:
`{ '@Foo': true, '@Bar': false } => { '@Foo': true, }` :| weird...